File: SIMESolution/SIME_Model/src/main/java/pe/edu/pucp/SIME/model/DTO/MatriculaAlumnoNuevoResponseDTO.java
package pe.edu.pucp.SIME.model.DTO;


import java.util.ArrayList;
import java.util.List;

public class MatriculaAlumnoNuevoResponseDTO {

    private int idAlumno;
    private int idMatriculaDetalle;
    private String mensaje;
    private List<PagoMatriculaDTO> pagos = new ArrayList<>();

    public int getIdAlumno() { return idAlumno; }
    public void setIdAlumno(int idAlumno) { this.idAlumno = idAlumno; }

    public int getIdMatriculaDetalle() { return idMatriculaDetalle; }
    public void setIdMatriculaDetalle(int idMatriculaDetalle) { this.idMatriculaDetalle = idMatriculaDetalle; }

    public String getMensaje() { return mensaje; }
    public void setMensaje(String mensaje) { this.mensaje = mensaje; }

    public List<PagoMatriculaDTO> getPagos() { return pagos; }
    public void setPagos(List<PagoMatriculaDTO> pagos) { this.pagos = pagos; }
}

File: SimeSoftSolution/SimeSoft/Models/Matriculas/MatriculaAlumnoNuevoResponseDto.cs
namespace SimeSoft.Models.Matriculas;

public class MatriculaAlumnoNuevoResponseDto
{
    public int IdAlumno { get; set; }
    public int IdMatriculaDetalle { get; set; }
    public string Mensaje { get; set; } = "";
    public List<PagoMatriculaResponseDto> Pagos { get; set; } = new();
}
